cicecore/cicedynB/general/ice_init.F90

```diff
@@ -2868,39 +2884,55 @@ subroutine set_state_var (nx_block, ny_block, &
          ! ice concentration/thickness
          !---------------------------------------------------------
 
-         if (trim(ice_data_type) == 'box2001'    .or. &
-             trim(ice_data_type) == 'smallblock' .or. &
-             trim(ice_data_type) == 'channel'    .or. &
-             trim(ice_data_type) == 'bigblock'   .or. &
-             trim(ice_data_type) == 'blockep5'   .or. &
-             trim(ice_data_type) == 'uniformp5'  .or. &
-             trim(ice_data_type) == 'gauss') then
+         if (trim(ice_data_conc) == 'p5') then
 
             hbar = c2  ! initial ice thickness
             do n = 1, ncat
                hinit(n) = c0
                ainit(n) = c0
                if (hbar > hin_max(n-1) .and. hbar < hin_max(n)) then
                   hinit(n) = hbar
-                  ainit(n) = p5 !echmod symm
+                  ainit(n) = p5
                endif
             enddo
 
-         elseif (trim(ice_data_type) == 'boxslotcyl' .or. &
-                 trim(ice_data_type) == 'medblocke'  .or. &
-                 trim(ice_data_type) == 'blocke') then
+         elseif (trim(ice_data_conc) == 'p8') then
+
+            hbar = c2  ! initial ice thickness
+            do n = 1, ncat
+               hinit(n) = c0
+               ainit(n) = c0
+               if (hbar > hin_max(n-1) .and. hbar < hin_max(n)) then
+                  hinit(n) = hbar
+                  ainit(n) = 0.8_dbl_kind
+               endif
+            enddo
+
+         elseif (trim(ice_data_conc) == 'p9') then
+
+            hbar = c2  ! initial ice thickness
+            do n = 1, ncat
+               hinit(n) = c0
+               ainit(n) = c0
+               if (hbar > hin_max(n-1) .and. hbar < hin_max(n)) then
+                  hinit(n) = hbar
+                  ainit(n) = 0.9_dbl_kind
+               endif
+            enddo
+
+         elseif (trim(ice_data_conc) == 'c1') then
          
             hbar = c1  ! initial ice thickness (1 m)
             do n = 1, ncat
                hinit(n) = c0
                ainit(n) = c0
                if (hbar > hin_max(n-1) .and. hbar < hin_max(n)) then
                   hinit(n) = hbar
-                  ainit(n) = c1 !echmod symm
+                  ainit(n) = c1
                endif
             enddo
          
-         else
+         elseif (trim(ice_data_conc) == 'default') then
```

Comment:
This has been the default, but it would be better to call it what it is, now that there are other options.  The ice concentration is parabolic in ice thickness space (the sum of aicen over all thicknesses is 1).

Reply:
I added parabolic as a valid option.